libs/vendored/crypto-relier/src/lib/deriver/deriver-utils.ts

```diff
@@ -16,29 +16,43 @@ export class DeriverUtils {
    * @param {string} bundle - String bundle to encrypt using the provided key
    * @returns {Promise}
    */
-  async encryptBundle(appPublicKeyJwk: string, bundle: string) {
-    const rawKey = jose.decodeJwt('.' + appPublicKeyJwk + '.');
-    const key = await jose.importJWK(rawKey);
+  encryptBundle(appPublicKeyJwk: string, bundle: string) {
```

Comment:
This is same as https://github.com/mozilla/fxa-crypto-relier/blob/c454131c4befaf007e6f245b1673f5078a34c5c5/src/deriver/DeriverUtils.js